### ISSUE TITLE
Ctrl D not working

### ISSUE THREAD

**Issue by SoiledPride** [PoE-TradeMacro/POE-TradeMacro#301]
Tried everything, all other commands work fine but not Ctrl D, the main reason people download this lol

**Comment by Eruyome** [comment]
Please describe your issue a bit more in detail. Is it just the tooltip not showing up? Maybe you're not running PoE in windowed mored?

**Comment by fraggynator** [comment]
I have the exact same issue.
Ctrl-C works, Ctrl-D does exactly nothing. 
AKH 1.1.25.01, POE is running windowed fullscreen.

**Comment by Eruyome** [comment]
Any chance you did the same as this guy https://github.com/PoE-TradeMacro/POE-TradeMacro/issues/306? Renamed the PathOfExile_x64.exe

**Comment by fraggynator** [comment]
Seen that, I didn't. The exe is PathOfExile_x64Steam.exe, figured this shouldn't be an issue since if that were the case, it had to be in FAQ (as it's standard for Steam installs)

**Comment by Eruyome** [comment]
Are any of the other hotkeys working?

**Comment by RespiceFinem** [comment]
I have this same issue; sometimes the script works without problem, other times it breaks and stops working entirely. Reloading the script or the game can sometimes fix it, but not always. Not seeing a consistent pattern to when it breaks, though.

**Comment by Eruyome** [comment]
Have you guys checked some older script versions to make sure this isn't something introduced with some specific version but a general issue happening with all of them?

**Comment by SoiledPride** [comment]
I was playing in full screen mode, didn't realise that made the difference.
Thanks.

**Comment by Eruyome** [comment]
So, one problem fixed, how about the others, I don't think you guys also ran the game in fullscreen. Unless somebody can tell me if this issue appears with any specific macro version or is independent of any version I can't really do anything.

**Comment by Eruyome** [comment]
Please test the new update and report back.

**Comment by XxNuB** [comment]
had same issue downloaded previous version to test it out now it is working fine... 
Issue I "had":
Ctrl+C Worked Ctrl+D Did not work... read this and tried downloading previous version, now script is working.

**Comment by Eruyome** [comment]
Could you clarify which version exactly works for you? v2.0.12 or one of the other previous ones?

**Comment by XxNuB** [comment]
v2.0.12 is currently the version that is working for me.  Not sure if this makes any sense or is in any way in connection to the issue. Currently Running PC off Large TV increased front size under display settings up to 150% (Win10).. Also my hard drive failed yesterday so I reinstalled Win10 on new hard drive with 15 other programs, games, add-ons, and communication software. (Including PoE Trade Macro) + (AHK). Also I believe updates for my drivers were also updating prior to issue.

I installed previous version that I knew worked for me 2 days ago v2.0.12. Scaled font back to default. Restarted my computer after all downloads were completed and drivers were updated.

I will try newest version today see if it is working now.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
